mkt/extensions/urls.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework.routers import SimpleRouter
+
+from mkt.extensions.views import ExtensionViewSet, ValidationViewSet
+
+extensions = SimpleRouter()
+extensions.register(r'validation', ValidationViewSet,
+                    base_name='extension-validation')
+extensions.register(r'extension', ExtensionViewSet)
```

Comment:
I thought about that, but then I would need to relocate `validation`, or forbid `validation` as a string, since we'll want `{prefix}/{slug}` for the detail API